Rakefile

```diff
@@ -0,0 +1,44 @@
+# Managed by modulesync - DO NOT EDIT
+# https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
+
+# Attempt to load voxpupuli-test (which pulls in puppetlabs_spec_helper),
+# otherwise attempt to load it directly.
+begin
+  require 'voxpupuli/test/rake'
+rescue LoadError
+  begin
+    require 'puppetlabs_spec_helper/rake_tasks'
+  rescue LoadError
+  end
```

Comment:
This looks old because e got rid of `puppetlabs_spec_helper`. Should have been part of https://github.com/voxpupuli/modulesync_config/commit/e326bebe6655f226c24a6fe20f3bd92802ee3781. https://github.com/voxpupuli/modulesync_config/pull/972